### ISSUE TITLE
Missed raises in match case blocks

### ISSUE THREAD

**Issue by jonyscathe** [jsh9/pydoclint#63]
The following function has the error: `Function `test` has a "Raises" section in the docstring, but there are not "raise" statements in the body Flake8(DOC502)`

It doesn't matter if the ValueError is within the catchall case or one of the actual cases.

```
def test(number: int) -> bool:
    """
    Test function.

    :param number: A number
    :type number: int
    :raises ValueError: When number is not 1 or 2
    :return: True if number is 1, False if 2
    :rtype: bool
    """
    match number:
        case 1:
            return True
        case 2:
            return False
        case _:
            raise ValueError('Number is not 1 or 2.')
```

**Comment by jonyscathe** [comment]
Oh, and relevant settings are:
```
skip-checking-short-docstrings = false
style = 'sphinx'
```

**Comment by jsh9** [comment]
Hi @jonyscathe , thank you for raising this issue!

It is indeed a bug in the pydoclint code, and I have fixed it in https://github.com/jsh9/pydoclint/pull/64.

**Comment by jsh9** [comment]
This change is included in the new published version, [0.2.0](https://github.com/jsh9/pydoclint/releases/tag/0.2.0).

**Comment by jonyscathe** [comment]
Awesome.
Thanks so much for resolving things so quickly.
